### PR TITLE
Add clean code tag

### LINKED ISSUE #6602: Add clean code as a tag on our nuget packages
Would be great to add clean code as a tag for our nuget packages. Currently we only have: `Roslyn`, `Analyzers`, `Refactoring` and `CodeAnalysis`. See: https://www.nuget.org/packages/SonarAnalyzer.CSharp/8.51.0.59060

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**
Fixes #6602

### CODE REVIEW

File: analyzers/packaging/SonarAnalyzer.CSharp.nuspec [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <releaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.52.0.0</releaseNotes>
     <language>en-US</language>
     <copyright>Copyright © 2015-2022 SonarSource SA</copyright>
-    <tags>Roslyn Analyzers Refactoring CodeAnalysis</tags>
+    <tags>Roslyn Analyzers Refactoring CodeAnalysis CleanCode</tags>
```
Comment (pavel-mikula-sonarsource) [inline analyzers/packaging/SonarAnalyzer.CSharp.nuspec:19]: What about this?
```suggestion
    <tags>Roslyn Analyzers Refactoring CodeAnalysis CleanCode Clean Code</tags>
```
There are only 8 packages that use "CleanCode" directly: https://www.nuget.org/packages?q=Tags%3A%22CleanCode%22

There are 168 packages that use "clean" (code, architecture, log, design, ....)
https://www.nuget.org/packages?q=Tags%3A%22Clean%22

I don't know how to search for "Clean" && "Code", unfortunately.

Maybe marketing has an answer to what we should use? As it's similar to SEO and they might already have a strategy about where we want to be.

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Channge itself looks good, I have a question about concrete tags

File: analyzers/packaging/SonarAnalyzer.CSharp.nuspec (hunk 1 above)
Comment (costin-zaharia-sonarsource) [reply analyzers/packaging/SonarAnalyzer.CSharp.nuspec:19]: I think there is nothing wrong with adding both "Clean" and  "Code". Will add them.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**